### PR TITLE
Write docs for air-gapped environments

### PR DESCRIPTION
**Pull request by mtojek**
Fixes: https://github.com/elastic/package-registry/issues/697

Preview: https://github.com/mtojek/package-storage/blob/697-air-gapped/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,3 +109,79 @@ docker.elastic.co/package-registry/distribution:7.9.0
...
+If you can route traffic to the public endpoint of EPR through a network gateway, there is a property in Kibana that
+can orchestrate it to use a proxy server:
+
+```yaml
+xpack.ingestManager.registryProxyUrl: your-nat-gateway.corp.net
+```
+
+Source: [Fleet overview](https://www.elastic.co/guide/en/fleet/7.13/fleet-overview.html#package-registry-intro)
+
+## Host own Elastic Package Registry
```
[the remark this answers is not in the document]
Comment (mtojek) [reply README.md:133]: Fixed

File: README.md [hunk 2]
```diff
@@ -109,3 +109,79 @@ docker.elastic.co/package-registry/distribution:7.9.0
...
+
+If routing traffic through a proxy server is not an option, you need to try the following approach - Package Storage instance must be deployed and hosted on-site as
+Docker container. Package Storage is a special distribution of the Package Registry which already includes packages.
+There are different distributions available:
+
+* production (recommended): `docker.elastic.co/package-registry/distribution:production` - stable, tested package revisions
+* staging: `docker.elastic.co/package-registry/distribution:staging` - package revisions ready for testing before release
+* snapshot: `docker.elastic.co/package-registry/distribution:snapshot` - package revisions updated on daily basis
```
[the remark this answers is not in the document]
Comment (mtojek) [reply README.md:143]: I added a short paragraph mentioning the release model.

File: README.md [hunk 3]
```diff
@@ -109,3 +109,85 @@ docker.elastic.co/package-registry/distribution:7.9.0
...
+
+5. (Optional) Define the internal healthcheck for the service as:
+
+```bash
+curl -f http://127.0.0.1:8080
+```
+
+## Connect Kibana to the hosted Package Registry
+
+There is a dedicated property in Kibana config to change the URL of the Package Registry's endpoint to a custom one,
```
Comment (ruflin) [inline README.md:188]: NIT: Should we mention that it requires a license?

**Review by ruflin — APPROVED** [review]
LGTM

**Comment by ruflin** [comment]
@mtojek I wonder if this docs should be in the package-registry repo instead of here?

**Comment by sorantis** [comment]
@ruflin @mtojek I'd go even further and suggest to add the final, supported version of this document to our official docs and accompany it with a blog post.

**Comment by mtojek** [comment]
> @mtojek I wonder if this docs should be in the package-registry repo instead of here?

Frankly speaking I had an internal discussion with myself on where to place this doc. I assumed it's strictly related to the package storage, so left it here.

> @ruflin @mtojek I'd go even further and suggest to add the final, supported version of this document to our official docs and accompany it with a blog post.

Not sure if we should do this as it's not supported/unofficial mode.

File: README.md (hunk 3 above)
Comment (mtojek) [reply README.md:191]: Fixed

**Comment by ruflin** [comment]
@sorantis As long as we do not support it we should also not promote it.
@mtojek "I had an internal discussion with myself" These are the best discussions 😆

**Comment by sorantis** [comment]
@ruflin If we don't promote it in a controlled manner, with a relevant disclaimer we won't get any feedback that can help us improve it. My comment about promoting it in the docs once we've reached the *final* and *supported* version of the document.

**Pull request merged**